tests/snapshots/output/grid,header.snapshot.txt

```diff
@@ -1,25 +1,25 @@
-───────┬────────────────────────────────────────────────────────────────────────
-       │ File: sample.rs
-───────┼────────────────────────────────────────────────────────────────────────
-       │ struct Rectangle {
```

Comment:
the vertical line is gone?

Reply:
That would be part of the resizing gutter change. When there aren't any line decorations, the vertical grid border is removed to save space (since there's nothing to separate).

I could add it back if the old behaviour is preferred, however.

Reply:
I would think that the vertical line is part of the grid and should be there if it's specified, but maybe @sharkdp can clarify

Reply:
I like this change. If there is nothing to display in the side-panel, I don't think there is any need for a vertical grid line.